### DISCUSSION TITLE
posts/%E6%AF%8F%E5%91%A8%E6%80%BB%E7%BB%93%E8%AF%BB%E4%B9%A6obsidian%E5%AE%A0%E7%89%A9%E5%98%89%E5%B9%B4%E5%8D%8E/

### DISCUSSION THREAD

**Discussion by giscus** [roninro/hugo-theme-puppet#45] [General]
# posts/%E6%AF%8F%E5%91%A8%E6%80%BB%E7%BB%93%E8%AF%BB%E4%B9%A6obsidian%E5%AE%A0%E7%89%A9%E5%98%89%E5%B9%B4%E5%8D%8E/

行动远比答案重要，只要你行动起来就赢了！

https://www.wangpangzier.com/posts/%E6%AF%8F%E5%91%A8%E6%80%BB%E7%BB%93%E8%AF%BB%E4%B9%A6obsidian%E5%AE%A0%E7%89%A9%E5%98%89%E5%B9%B4%E5%8D%8E/

**Comment by DayuGuo** [comment]
好多狗狗 哈哈哈
